### DISCUSSION TITLE
Multi-valued email headers

### DISCUSSION THREAD

**Discussion by agroebe** [O365/python-o365#788] [Q&A]
Hello,

I am working on a service that uses O365 for email automation and I need to ask a specific question about how this library handles multi-valued HTTP headers.

Consider a scenario when I request HTTP headers for messages as such
`get_messages(query=mb.q().select('internet_message_headers'))`

I am wondering how header dictionaries store multi-valued headers. 

Are they comma separated as a string, like so:
```
{
    'name': 'Some-Header-Name',
    'value': 'first header val, second header val, third header val'
}
```

OR are they appended to a list, like so:
```
{
    'name': 'Some-Header-Name',
    'value': [
        'first header val',
        'second header val',
        'third header val'
    ]
}
```

**Comment by alejcas** [comment]
Check the MS Graph docs under the internet message headers article.

That will answer your question.
